### PR TITLE
Add explicit swift_version

### PR DESCRIPTION
**Pull request by pixelmatrix**
If you pod install on a new project in Xcode 10 it will default to Swift version of 4.2, which made some breaking changes that this library needs to address. This will fix the pod for people installing until the library supports 4.2.

### CODE REVIEW

**Comment by Ybrin** [comment]
@pixelmatrix I think we are using some 4.1 features. But I'm not entirely sure. Is there a reason you set it to 4.0?

**Comment by Ybrin** [comment]
If it does it's fine for me. Just wanted to know the reason. 4.1 is really
a weird version...

**Comment by pixelmatrix** [comment]
@rogerluan Good catch. I ran the tests locally and then I forgot to follow up on this PR. Didn't realize there was a .swift-version file, but since it's deprecated, I removed it just now.

**Comment by rogerluan** [comment]
Looks like removing that file broke something in the CI environment 🤔

**Comment by pixelmatrix** [comment]
Yep, looks like there's something in the Travis config that relies on that file. @Ybrin can we remove that portion of the config now?

More on the deprecation here:
https://blog.cocoapods.org/CocoaPods-1.4.0/

**Comment by Ybrin** [comment]
Yes I was using .swift-version to install the swift compiler on the Linux test system. I guess we will have to manually specify it now in the travis config file.

**Comment by pixelmatrix** [comment]
@Ybrin Maybe it can parse the Podspec and read the swift version from there?

**Comment by pixelmatrix** [comment]
Alternatively we can leave the swift-version file for now and just make sure it matches. Might be just pushing the pain down the road though.

**Comment by Ybrin** [comment]
Parsing the podspec would be ideal. But I've never done this before. If you know how, fell free to implement it, even if it's a little bit of an overkill, I appreciate it 😉

File: Example/Podfile.lock [hunk 1]
```diff
@@ -7,18 +7,18 @@ PODS:
...
-    - Web3/HTTPExtension (= 0.3.0)
-  - Web3/ContractABI (0.3.0):
+  - Web3 (0.3.1):
+    - Web3/Core (= 0.3.1)
+    - Web3/HTTPExtension (= 0.3.1)
+  - Web3/ContractABI (0.3.1):
     - Web3/Core
-  - Web3/Core (0.3.0):
+  - Web3/Core (0.3.1):
     - BigInt (~> 3.1)
```
Comment (amaliazulfia) [inline Example/Podfile.lock:16]: WTF arre you doing

**Pull request closed without merging**